core/src/main/java/org/jenkins/ui/symbol/Symbol.java

```diff
@@ -94,7 +94,6 @@ private static String loadSymbol(String namespace, String name) {
                      .replaceAll("(class=\").*?(\")", "")
                      .replaceAll("(tooltip=\").*?(\")", "")
                      .replaceAll("(data-html-tooltip=\").*?(\")", "")
-                     .replaceAll("(id=\").*?(\")", "")
```

Comment:
I think that can lead to a duplicated id for the svg itself.
Assume the svg has an id at the root `<svg id="symbolid" .../>`
Now I want to use that svg with 
`<l:icon sec="mysymbol" id="myid"/>`
The result will be 2 id attributes I think.

Reply:
it could but why would the svg have a symbol at the root baked into it?

Otherwise to fix it this string replace hack needs replacing with a dom library that only operates on the svg tag and not children.